### PR TITLE
Add OSDU Ontology 

### PR DESCRIPTION
**Pull request by neda-abolhassani**
Please add OSDU ontology to your repo. This is for registering a unique namespace with the prefix osdu.

### CODE REVIEW

**Comment by dgarijo** [comment]
@neda-abolhassani,
you are missing the redirection file to your target ontology for your w3id (htaccess).  Can you please add it in your PR?

File: osdu/.htaccess [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^$ https://github.com/Accenture/OSDU-Ontology [R=302,L]
```
Comment (davidlehn) [inline osdu/.htaccess:3]: Note that this is pointing at your github page, not the plain TTL file.  So if URLs like https://w3id.org/osdu#xxx are dereferenced, they would just point at the github UI.  Is that ok?

File: osdu/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+The OSDU Ontology is licensed under the Apache License 2.0 - see [License](./License).
+
+
+## Homepage
+[OSDU Ontology](https://github.com/Accenture/OSDU-Ontology/)
+
+## Vocabulary Usage
+
+    @prefix osdu: <https://w3id.org/osdu#>
```
Comment (davidlehn) [inline osdu/README.md:14]: https://github.com/Accenture/OSDU-Ontology/blob/main/ttl/OSDU.ttl is using https://w3id.org/OSDU# vs https://w3id.org/osdu# used here and the directory itself..  This service is case sensitive.

File: osdu/.htaccess (hunk 1 above)
Comment (neda-abolhassani) [reply osdu/.htaccess:3]: I'll fix it.

File: osdu/.htaccess [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^$ https://github.com/Accenture/OSDU-Ontology/blob/main/ttl/OSDU.ttl [R=302,L]
```
Comment (davidlehn) [inline osdu/.htaccess:3]: This is still pointing to the github UI.  You might consider the "raw" link: https://raw.githubusercontent.com/Accenture/OSDU-Ontology/main/ttl/OSDU.ttl.  Or setup your project to publish to github pages and use a link from there.

Comment (neda-abolhassani) [reply osdu/.htaccess]: I have changed the link to https://accenture.github.io/OSDU-Ontology/

**Pull request merged**